WordPress/gutenberg
# Fix: add 'application/zip' accept header

The export call is missing the `application/zip` accept header. See https://github.com/Automattic/wp-calypso/issues/59527 for further information.



## What?

This PR add an `Accept: application/zip` to the export function.

## Why?
When an accept header of `*/*` is sent the browser can't download the file. This happens in production environments.


## How?


## Testing Instructions
1. Ensure to run a local production site at http://localhost:8888/wp-admin/ (see instructions here: https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/getting-started-with-code-contribution.md)
2. Select Appearence -> Editor
3. Select the top right menu
4. Select "Export"
5. Ensure that a valid .zip is exported
